### DISCUSSION TITLE
RF-Gateway HM-CFG-USB (JEQ0534706) not connected

### DISCUSSION THREAD

**Discussion by Moach2000** [OpenCCU/OpenCCU#2031] [🇩🇪 Fragen&Antworten]
Hallo!
Habe heute versucht, auf meinem Raspi 4 / 8GB mit Home Assistant
 Home Assistant 2022.9.1 Supervisor 2022.10.0 Operating System 9.2 Frontend 20220907.0 - latest 
mittels Addon und einem Stick HM-CFG-USB2
(Log im Add-on --> Identifying Homematic RF-Hardware: .HmRF: HM-CFG-USB-2/USB, HmIP: n/a, OK)
einen Heizkörperthermostat anzulernen.

Log Add-On (komplett):
Mounting /data as /usr/local (Home Assistant Add-On): OK
Starting watchdog...
Identifying host system: oci, OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.6, OK
Identifying Homematic RF-Hardware: .HmRF: HM-CFG-USB-2/USB, HmIP: n/a, OK
Updating Homematic RF-Hardware: no GPIO/USB connected RF-hardware found
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: not required
Starting rfd: .OK
Starting HMIPServer: ......OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.65.11.20221005 (raspmatic_oci_arm64)

Leider kommt immer wieder ein Watch-Dog Fehler
(Log RapberryMatic --> 

WatchDog: connect-HM-CFG-USB | ausgelöst | RF-Gateway HM-CFG-USB (JEQ0534706) not connected | ausgelöst)

Hab ich was in der Anleitung falsch verstanden?

Danke und 
Grüße
Chris

**Comment by andipilz** [comment]
Hallo Chris und Team,
ich habe ein sehr ähnlich gelagertes Problem: ich nutze den HM-CFG-USB-2 mit RaspberryMatic, im Container von Home Assistant. Ich stelle fest, dass das "pass through" des Sticks immer wieder verschwindet, da das Host system (Ubuntu 22.04 LTS) immer wieder die DeviseID ändert und damit das Durchreichen nicht mehr funktioniert, bis ich e.g. das Pas Through lösche und neu aktiviere. Ein regelmäßiges `lsusb` hilft.
Suche immer noch nach kreativen Ideen, wie ich am Host System die Device ID festschreiben kann.
Dank und Gruß
Andreas
